recipes/cython_gsl/meta.yaml

```diff
@@ -1,19 +1,21 @@
-{% set name = "CythonGSL" %}
+{% set name = "cython_gsl" %}
```

Comment:
The module import is `cython_gsl`, but the name is still `CythonGSL`.
See  https://github.com/twiecki/CythonGSL/blob/master/setup.py#L5

(I am worried about users searching this and not finding what they want.)